wifiphisher/data/phishing-pages/oauth-login/html/nointernet.html

```diff
@@ -0,0 +1,2 @@
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<p>User Didn't Sign In</p>
```

Comment:
Once the user press the cancel button on the js popup he will be redirected to a new page and this message will be showed.
Of course this one is used only to force the user to connect because once he sees this message then he'll think that the provided service is real and he'll know that there is no other way to connect besides from this.